python/doc/examples/probabilistic_modeling/random_vectors/plot_ratio_of_uniforms_exploitation.py

```diff
@@ -0,0 +1,455 @@
+#
+# The following function draws the boundary :math:`\partial \varphi(A_{f,r})` and the bounding box
+# :math:`\partial \tilde{A}_{f,r}` in the :math:`(x,y)`-space.
+
+
+def draw_RatioOfUniforms_XY(ratio_algo, nPoints):
+    r = ratio_algo.getR()
+    range_logF = ratio_algo.getRange()
+    xInf = range_logF.getLowerBound()[0]
+    xSup = range_logF.getUpperBound()[0]
+    uSup = ratio_algo.getSupU()
+    vInf = ratio_algo.getInfV()[0]
+    vSup = ratio_algo.getSupV()[0]
+    logF = ratio_algo.getLogUnscaledPDF()
+
+    # \partial varpĥi(A_{f,r}) domain
+    eps = 1e-10
+    step = (xSup - xInf - 2.0 * eps) / (nPoints - 1)
+    data_XY = ot.Sample(nPoints, 2)
+    for i in range(nPoints):
+        x = xInf + eps + i * step
+        val_F = exp(logF([x])[0])
+        data_XY[i, 0] = x
+        data_XY[i, 1] = val_F ** (1.0 / (1.0 + r))
+    g = ot.Graph("", "x", "y", True, "lower center")
+    data_A = ot.Curve(data_XY)
+    data_A.setColor("purple")
+    g.add(data_A)
+
+    # \partial varphi(\tilde{A}_{f,r}) bounding box
+    leg = ot.Description()
+    leg.add(r"boundary of $\varphi(A_{f,r})$")
+    boundary_1_fct = ot.SymbolicFunction(
+        "x", "(" + str(vSup) + "/x)^(1/" + str(r) + ")"
+    )
+    if vSup / (uSup**r) < xSup:
+        boundary_1 = boundary_1_fct.draw(vSup / (uSup**r), xSup, 101)
+        boundary_1.setColors(["blue"])
+        leg.add(r"$\varphi(\mathcal{F}_1)$")
+        g.add(boundary_1)
+    if vInf / (uSup**r) < uSup:
+        boundary_2_fct = ot.SymbolicFunction(
+            "x", "(" + str(vInf) + "/x)^(1/" + str(r) + ")"
+        )
+        boundary_2 = boundary_2_fct.draw(xInf, vInf / (uSup**r), 101)
+        boundary_2.setColors(["red"])
+        leg.add(r"$\varphi(\mathcal{F}_2)$")
+        g.add(boundary_2)
+    boundary_3 = ot.Curve([[vInf / (uSup**r), uSup], [vSup / (uSup**r), uSup]])
+    boundary_3.setColor("green")
+    g.add(boundary_3)
+    leg.add(r"$\varphi(\mathcal{F}_3)$")
+    g.setLegends(leg)
+
+    return g
+
+
+# %%
+# Illustration.
+# -------------
+# We create the function :math:`\log f` defined by:
+#
+# .. math::
+#    :label: robertEx
+#
+#    \log f(x) = \log(\cos(x)) + x,  \quad  x \in \left[-\dfrac{\pi}{2}, \dfrac{\pi}{2}\right]
+#
+# and the :class:`~openturns.experimental.RatioOfUniforms`.
+
+
+log_UnscaledPDF = ot.SymbolicFunction("x", "log(cos(x)) + x")
+range_PDF = ot.Interval(-pi / 2.0, pi / 2.0)
+ratio_Algo = otexp.RatioOfUniforms(log_UnscaledPDF, range_PDF)
+
+# %%
+# We draw the boundary :math:`\partial A_{f,r}` and the bounding box :math:`\partial \tilde{A}_{f,r}`
+# in the :math:`(u,v)`-space.
+g_UV = draw_RatioOfUniforms_UV(ratio_Algo)
+g_UV.setTitle(
+    rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+)
+
+# %%
+# We draw the boundary :math:`\partial \varphi(A_{f,r})` and the bounding box :math:`\partial \varphi(\tilde{A}_{f,r})`
+# in the :math:`(x,y)`-space.
+# sphinx_gallery_thumbnail_number = 1
```

Comment:
go to the line to avoid it to follow the previous sentence

Reply:
yes...